zebra-rpc/src/methods/tests/snapshot/snapshots/snapshot_rpc_submit_block_invalid@mainnet_10.snap

```diff
@@ -0,0 +1,7 @@
+---
+source: zebra-rpc/src/methods/tests/snapshot/get_block_template_rpcs.rs
+expression: submit_block_response
+---
+{
+  "Ok": "rejected"
```

Comment:
This seems like a bug, the specified response format is:
```
"rejected"
```

https://zcash.github.io/rpc/submitblock.html

Reply:
It wasn't unwrapping the result in the snapshot test, updated in [`cf4e937`](https://github.com/ZcashFoundation/zebra/pull/5526/commits/cf4e9376e4af691f2dc1183f53b1c95cdb91bcd5)